lib/gnucash/invoice/runner.rb

```diff
@@ -22,38 +42,95 @@ def run(argv)
           end
 
           # Show known invoices
+          puts INVOICE_LIST_HDR
+          puts INVOICE_LIST_FMT % [
+                 'Invoice ID', 'Customer', 'Reference', 'Opened at', 'Posted at',
+                 'Due at'
+               ]
+          puts INVOICE_LIST_HDR
```

Comment:
I don't think these two should be constants.

I think we should add [teminal-table](https://github.com/tj/terminal-table/) gem instead and use it here:

``` ruby
puts Terminal::Table.new({
  :headings => ['Invoice ID', 'Customer', 'Reference', 'Opened at', 'Posted at', 'Due at']
  :rows => Invoice.all.map { |invoice| [invoice.id, ...] }
})
```